Sources/ForageSDK/ForageSDK.swift

```diff
@@ -47,11 +63,6 @@ public class ForageSDK {
     public struct Config {
         var merchantID: String
         var sessionToken: String
-
-        public init(merchantID: String, sessionToken: String) {
-            self.merchantID = merchantID
-            self.sessionToken = sessionToken
-        }
```

Comment:
Swift structs get "member-wise initializers" (an `init` method that has each property as a parameter) by default, so we don't need to explicitly define this. 
![CleanShot 2024-08-15 at 12 20 41@2x](https://github.com/user-attachments/assets/146b356b-44ed-4b95-ba48-6f982afda695)
![CleanShot 2024-08-15 at 12 21 20@2x](https://github.com/user-attachments/assets/1e96ef66-e89f-4787-a0aa-e33403f35bdf)